thirtyinch-lint/src/main/kotlin/net/grandcentrix/thirtyinch/lint/detector/MissingViewInThirtyInchDetector.kt

```diff
@@ -0,0 +1,57 @@
+package net.grandcentrix.thirtyinch.lint.detector
+
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.intellij.psi.PsiClass
+import com.intellij.psi.PsiClassType
+import com.intellij.psi.PsiType
+import net.grandcentrix.thirtyinch.lint.TiIssue.MissingView
+import org.jetbrains.uast.UClass
+
+private const val TI_VIEW_FQ = "net.grandcentrix.thirtyinch.TiView"
+private const val PROVIDE_VIEW_METHOD = "provideView"
+private val TI_CLASS_NAMES = listOf(
+        "net.grandcentrix.thirtyinch.TiActivity",
+        "net.grandcentrix.thirtyinch.TiFragment"
+)
+
+class MissingViewInThirtyInchDetector : BaseMissingViewDetector() {
+    companion object {
+        val ISSUE = MissingView.asLintIssue(
+                MissingViewInThirtyInchDetector::class.java,
+                "When using ThirtyInch, a class extending TiActivity, TiFragment or CompositeActivity " +
+                        "has to implement the TiView interface associated with it in its signature, " +
+                        "or implement `provideView()` instead to override this default behaviour."
+        )
+    }
+
+    override fun applicableSuperClasses() = TI_CLASS_NAMES
+
+    override val issue: Issue; get() = ISSUE
```

Comment:
Why is here an semicolon?
Should be removed. See also [this style guide](https://kotlinlang.org/docs/reference/coding-conventions.html#property-formatting).

Reply:
This semicolon is needed to keep it all in one line. But I removed the getter overriding completely.